labs/electronic-toll-collection-system.md

```diff
@@ -0,0 +1,54 @@
+
+The government information is very sensible, and the message ETC is a root information to create invoice where federal institutions receive incomes and after they need to make a payments to operators companies by services of leasing from IoT devices, and adminstration payments too.
+
+As we can see, ETC message is very important to make concilian information to differents participants with some confusions about charges made or authorities whom receive incomes from differents operators companies.
+
+Hyperledger fabric has a important role to keep and save information with principal objective message ETC, acomplish with this, inmutability, integrity and to government and some private companies, they can have control about their own information.
+
+With the foregoing, the acquisition of the NEAR Token by the client is determined to carry out requests to extract information directly from the private blockchain, said fee will be to meet transaction expenses for the creation of receipt delivery receipt; a web portal will be created where the transfer of funds can be made, which will be able to interact with the smart contract developed in NEAR.
+
+Once the transfer of funds for the acquisition of information from the company/operator, a temporary channel will be created that will expose the corresponding information of the applicant, said information extracted from the private blockchain network.
+
+Continuing with the flow of information transparency, said funds transferred through the NEAR token will continue to be transparent for the client, since they will be in a public network, said protocol was selected, since the cost of transactions is lower. to others such as ETHEREUM, in addition to the use of the Proof Of Stake (PoS) protocol, making transactions per second (TPS) more agile and faster.
+
+The benefits of the solution are several, where the most important are:
+
+1.- Transparency and Integrity in the information collected from the costs of the operators,
+
+2.- If there is a loss of communication, it is avoided having to make unnecessary queries to the Ledger or DB engine, since the management of queues applies fault tolerance concepts.
+
+3.- It is guaranteed that the interoperability tags are delivered to the operator that belongs to it, removing the SFTP servers.
+
+4.- Information may be reconciled by operators, if the figures do not coincide with what is requested by the authority.
+
+5.- Information on control figures can be uploaded every month end.
+
+
+![INTER-OPERABILIDAD-WorkFlow_ ETC_ NEAR - GATEWAY (4)](https://user-images.githubusercontent.com/11085724/169954095-848c83e5-613a-4297-8376-84a7a0866d75.jpg)
```

Comment:
It would be good to include the image in the `labs/images` folder

Reply:
gr8, I've added the images into labs images path too. tnx